pkg/apis/networking/v1/utils.go

```diff
@@ -67,11 +67,23 @@ func GetNetworkMode(networkObj *Network) NetworkMode {
 		if len(networkObj.Spec.Mode) == 0 {
 			return NetworkModeVxlan
 		}
+	case NetworkTypeGlobalBGP:
+		if len(networkObj.Spec.Mode) == 0 {
+			return NetworkModeGlobalBGP
+		}
 	}
 
 	return networkObj.Spec.Mode
 }
 
+func IsGlobalNetwork(networkObj *Network) bool {
```

Comment:
What is `global` meaning here? IMO, globalBGP is not completely same as overlay. Also the capacity validation in webhook is not only for overlay and globalBGP, I believe we can do same validation on underlay network. So how about removing these two functions?

Reply:
how about renaming it `IsUniqueNetwork`? seems more proper

Reply:
It's renamed to `IsGlobalUniqueNetwork`.